.github/workflows/cherry-picker.yml

```diff
@@ -0,0 +1,53 @@
+name: Cherry pick
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: ["closed"]
+
+permissions:
+  contents: read
```

Comment:
can we create branch with read permission?

Reply:
Thank you for pointing that out. I should have update the permissions in the workflow to ensure that the action can create and push branches. The permissions should be set to `contents: write` and `pull-requests: write`.

Reply:
@Manoramsharma thanks for you work, but I think this's not a good issue for begginer(this one need a lot of collaborations with the Envoyproxy org admin).